Demo/ViewController.swift

```diff
@@ -206,9 +195,8 @@ class ViewController: UITableViewController {
             navigationController?.pushViewController(viewController, animated: true)
 
         default:
-            let viewController = HyperwalletUI.shared.listTransferMethodController()
-
-            navigationController?.pushViewController(viewController, animated: true)
+            let coordinator = HyperwalletUI.shared.listTransferMethodCoordinator(parentController: self)
+            coordinator.navigate()
         }
     }
```

Comment:
Remove createTransferMethodHandler and implement didFlowComplete func

Reply:
Added didFlowComplete func